pkg/kyma/cmd/provision/minikube.go

```diff
@@ -129,30 +130,34 @@ func (o *MinikubeOptions) Run() error {
 	}
 	s.Successf("Minikube config initialized")
 
-	s = o.NewStep(fmt.Sprintf("Waiting for minikube to be up and running"))
+	s = o.NewStep(fmt.Sprintf("Create minikube instance"))
+	s.Status("Start minikube")
 	err = startMinikube(o)
 	if err != nil {
 		s.Failure()
 		return err
 	}
 
-	err = waitForMinikubeToBeUp()
+	s.Status("Await minikube to be up and running")
+	err = waitForMinikubeToBeUp(s)
 	if err != nil {
 		s.Failure()
 		return err
 	}
 
+	s.Status("Create default cluster role")
 	err = createClusterRoleBinding()
 	if err != nil {
 		s.Failure()
 		return err
 	}
 
-	err = internal.WaitForPod("kube-system", "k8s-app", "kube-dns")
-	if err != nil {
-		s.Failure()
-		return err
-	}
+	//s.Status("Await kube-dns to be up and running")
```

Comment:
Can you please decide if it is needed or not and then keep or delete the code.

Reply:
What I'm sure is it has changed in new minikube. What I can do is write it the same way installation scripts are written (they work on labels which haven't changed).

Reply:
I thought the problem was with renaming in minikube, but the problem was with IsPodReady function. Fix is en route.

Reply:
jihaa, thanks